tpu_inference/distributed/tpu_connector_local.py

```diff
@@ -715,6 +829,11 @@ def build_connector_meta(
                         f"total_tokens={len(tracker.token_ids)}, "
                         f"total_blocks={len(tracker.block_ids)}")
 
+            # NOTE(jcgu): why only save op here?
+            if req_id in self.load_specs:
```

Comment:
we can remove this warning message, since the load_spec is only cleared in handling scheduler_output.finished_req_ids L698 in this PR